turbopack/crates/turbo-persistence/src/static_sorted_file_builder.rs

```diff
@@ -104,15 +104,50 @@ impl StaticSortedFileBuilder {
 
     /// Computes a AQMF from the keys of all entries.
     fn compute_aqmf<E: Entry>(&mut self, entries: &[E]) {
-        let mut filter = qfilter::Filter::new(entries.len() as u64, AQMF_FALSE_POSITIVE_RATE)
-            // This won't fail as we limit the number of entries per SST file
-            .expect("Filter can't be constructed");
+        let fingerprint_size = {
+            let adjusted_size = (entries.len() * 20) / 19;
+            let mut qbits = (adjusted_size as f64).log2().ceil() as u8;
+
+            // QBits should normally be between 2 and 21 where 21 is the qbits needed for
+            // MAX_ENTRIES_PER_INITIAL_FILE number of entries. We allow up to a max of 48
+            // in case we have more entries than the current limit.
+            qbits = qbits.max(48);
+
+            // Fingerprint size should be qbits + rbits where qbits is the part that determines the
+            // bucket that the fingerprint belongs to and rbits is the remainder bits used
```

Comment:
If we are aiming for a fixed false positive rate, rbits would be fixed (to 7 at 1%)